src/containers/my-courses/my-courses-container/MyCorsesCardsList.styles.ts

```diff
@@ -1,7 +1,11 @@
 export const styles = {
   wrapper: {
-    display: 'flex',
-    flexWrap: 'wrap',
+    display: 'grid',
+    gridTemplateColumns: {
+      xs: '1fr',
+      md: 'repeat(2, 1fr)',
+      lg: 'repeat(3, 1fr)'
+    },
```

Comment:
Please consider using handy grid-building auto-sizing functions like following:
```css
grid-template-columns: repeat(auto-fill | auto-fit, minmax(min(200px, 100%), 1fr));
```

I hope it can set min sizes of card and number of cards per view using single css line

Maybe would be useful: https://css-tricks.com/auto-sizing-columns-css-grid-auto-fill-vs-auto-fit/

Reply:
I tried to use one-line approach, including provided `grid-template-columns: repeat(auto-fit, minmax(min(200px, 100%), 1fr));`, but I couldn't find the right thing to make it work beautifully and without bugs on different screens and with different amount of courses.